lib/go-tc/server_server_capability.go

```diff
@@ -27,6 +27,11 @@ type ServerServerCapability struct {
 	ServerCapability *string    `json:"serverCapability" db:"server_capability"`
 }
 
+type MultipleServerCapabilities struct {
+	ServerID         *int     `json:"serverId" db:"server"`
```

Comment:
I don't think this property is optional, so it need not be a pointer.

Also; nit but `serverId` should be `serverID` to be properly `camelCased`

Reply:
`serverId` is for the json text, it is camelCased when you consider it as a go variable. I removed the pointer

Reply:
Yeah I know that's the JSON name. We have a bad habit of not capitalizing full initialisms in our JSON property names - so this is consistent with most of the API and you can leave it alone, but for whatever it's worth, I'd make it `serverID`